src/classes/EscalationTrigger.cls

```diff
@@ -193,6 +194,24 @@ public with sharing class EscalationTrigger extends BaseTrigger {
 	}
 
 	/**
+	* Assign default owner to an escalation
+	*/
+	@TestVisible
+	private void assignDefaultOwner() {
+		if (!this.isInsert) {
+			return;
+		}
+
+		rh_escal__EscalationRouter__c er = rh_escal__EscalationRouter__c.getValues(DEFAULT_ESCAL_ROUTER);
```

Comment:
If that's the case, pull all of this out and move it into the impl package.  We don't want to auto assign a default and not be able to assign one per record type since the trigger order is not guaranteed.